src/react/projects/spark-core-react/src/SprkPromo/SprkPromo.js

```diff
@@ -0,0 +1,233 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { SprkLink, SprkButton } from '@sparkdesignsystem/spark-core-react';
+
+const SprkPromo = (props) => {
+  const { 
+    title,
+    subtitle,
+    additionalClasses,
+    additionalClassesContent,
+    cta,
+    ctaText,
+    ctaHref,
+    ctaAnalytics,
+    ctaIdString,
+    imgSrc,
+    imgAlt,
+    imgLinkHref,
+    imgLinkAnalytics,
+    imgLinkIdString,
+    additionalClassesImgLink,
+    idString,
+    isFlag,
+    mediaRev,
+    hasBorder,
+    children,
+    ...other
+  } = props;
+  
+  return (
+    <div
+      className={
+        classnames(
```

Comment:
I typically see this done outside of the component.

`className={classNames}` here,

and somewhere else

```
    const classNames = classnames(
        'sprk-c-Dictionary',
        additionalClasses, {
            'sprk-c-Dictionary--striped': variant === 'striped'
        }
    );
```

Reply:
I like your way. I think it will make the component look a lot cleaner.